.github/workflows/build-docker.yml

```diff
@@ -65,7 +65,7 @@ jobs:
       - name: Check out Celeritas
         uses: actions/checkout@v3
         with:
-          fetch-depth: 255
+          fetch-depth: 383
```

Comment:
What is the 'cost'/'loss' if this number is too low?

Reply:
Cost is finding and downloading extra Git objects, potentially slowing down the CI. I didn't think we'd ever reach 255 TBH.... we should've released 0.5 months ago 😅instead of waiting for "just around the corner" optical physicsl.

Reply:
What about the cost/loss of functionality in the reverse case, i.e. in the extreme:
```
fetch-depth: 1
```
?

Reply:
Then we get CI errors like this because it can't detect the version because the tag isn't within the last 1 depth: https://github.com/celeritas-project/celeritas/actions/runs/10724334134/job/29739743532